docs/src/about/FAQ.md

```diff
@@ -32,16 +35,29 @@ You can use test tokens in the same way you would use value-bearing tokens.
 
 For example with [PolkadotJS Apps](https://polkadot.js.org/apps/) you can create a transaction on behalf of your account, generate a signature with Signer and submit it to the network. All of this without keys ever leaving your offline device.
 
-### How do I keep my keys secure?
+## Networks
 
-Signer is a safe way to use your keys. However, that alone won't be enough to keep your keys secure. Devices break and get lost. This is why we always recommend to backup your seed phrases and derivation paths on paper. We are such big fans of paper backups that we even support a special tool to power your paper backup game by splitting your backups into shards called [Banana Split](https://bs.parity.io/).
+### How can I update metadata version for a network?
 
-### How do I know I am not interacting with malicious apps or actors?
+Please go to [Metadata Update Portal](https://metadata.parity.io/) and select network you need to update. Scan multipart QR code the same way you scan a transaction QR code. 
```

Comment:
we need to tell here that this could be done in a decentralized way, otherwise we'll get a lot more questions "why is this centralized?" and "how do I update it without you then?" and also "how do I update network-nobody-else-knows-about?"